upgrading.md

```diff
@@ -17,7 +17,15 @@ command from the root of your app (the same directory that contains the
 $ flutter upgrade
```

Comment:
Does this also run `flutter packages upgrade` ?

Or is it now purely scoped to just upgrading the flutter SDK associated with this app?